MessageTerminal/MessageTerminal.cxx

```diff
@@ -7,6 +7,7 @@
 #include "TestStandCommHost.h"
 
 #include "shared_defs.h"
+#include "SerialResult.h"
```

Comment:
One is snake case, the other is camel, does this signify something?

Reply:
Uhh, poor consistency?
I can fix that

Reply:
Actually, I'm gonna change it in a future PR cause otherwise I'll create conflicts with myself